### DISCUSSION TITLE
Phone variable non clickable

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#15676] [How to] [answered]
Trying to make phone number printed in email template via var clickable, but non method is working. Is there any trick on that?

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
try this
https://github.com/YetiForceCompany/YetiForceCRM/commit/c4625ae1e35e16998796a623b4b8e57195f89ab2

**Reply by s2b-git** [reply]
Amazing. Thank you
